ui-v2/app/templates/dc/nodes/-services.hbs

```diff
@@ -15,7 +15,10 @@
         {{/block-slot}}
         {{#block-slot 'row'}}
             <td data-test-service-name="{{item.Service}}">
-              <a href={{href-to 'dc.services.show' item.Service }}>{{item.Service}}{{#if (not-eq item.ID item.Service) }} <em data-test-service-id="{{item.ID}}">({{item.ID}})</em>{{/if}}</a>
+              <a href={{href-to 'dc.services.show' item.Service}}>
+                <span data-test-external-source="{{service/external-source item}}" style="background-image: {{css-var (concat '--' (service/external-source item) '-color-svg') 'none'}}"></span>
```

Comment:
The icons themselves are inline CSS. 2 reasons for this:

1. I want the icons themselves to potentially be configurable from the HTML (not CSS - so no `.external-source-consul`, `.external-source-nomad` etc).
2. The logo itself is actually a decorative/decorational 'thing' - its a design decision that we show an icon not just the word 'consul' - therefore it's done via CSS not an inline SVG. I did um and ahh about this a bit, but this is the side I fell on.